standalone/node/src/service.rs

```diff
@@ -293,15 +268,14 @@ pub fn new_full(
 		);
 
 		let relayer_params = webb_relayer_gadget::WebbRelayerParams {
-			local_keystore: keystore_container.local_keystore(),
+			local_keystore: Some(keystore_container.local_keystore()),
```

Comment:
if it is always there, there is no need to wrap it into Option::Some, you can change the `WebbRelayerParams` to take that value directly.